ui-tests/notebooks/Subitems.ipynb

```diff
@@ -67,6 +68,13 @@
     "m1.add(layer1)\n",
     "m1.add(layer2)"
    ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Ok bare with me, our test does not like executing empty cells, it makes the execution time out :D You need to remove the empty cell from the notebook